### ISSUE TITLE
Indicators for main events in the application  

### ISSUE THREAD

**Issue by mistryishan25** [aditicsalunkhe/Simplii-Group74-2023#15]
**Is your feature request related to a problem? Please describe.**
Even though the `send_requests` feature works, there is no indicator to notify the user if they have any requests or pending approvals.

**Describe the solution you'd like**
Adding a box element at the right of the card that keeps the count of these events would be a nice add-on.

**Describe alternatives you've considered**
We can start with a basic idea - fetching the count of the events in the list of these features and adding them as text right next to the heading.

**Comment by WangWangIsDog** [comment]
I think that`s a good idea since it can greatly improve user experience

**Comment by aditicsalunkhe** [comment]
Yes, it sounds like a great idea. But, where do you plan to put this count? I assume it is like a notification count that we get  on the top right corner of the application. We can put it on the top right of the "Friends" link and then in top right corner of "Pending Approvals" link. Let me know if it makes sense.
